package.json

```
@@ -6,6 +6,13 @@
     "yarn": ">=1 <2"
   },
   "scripts": {
+    "start:frontend": "npm run start -w frontend",
+    "start:backend": "npm run build -w shared && npm run start:dev -w backend",
+    "migrate": "npm run migrate -w backend",
```

Please, don't use both the `npm` & `yarn` in the same `package.json` :pray: 

--- 

`shared` has its own `start:dev` script. Also, we shouldn't run the shared package as part of some other app start script

```suggestion
    "start:frontend": "yarn start -w frontend",
    "start:backend": "yarn start:dev -w backend",
    "start:shared": "yarn start:dev -w shared",
```